src/components/color-picker/color-picker.tsx

```diff
@@ -87,6 +101,23 @@ export class ColorPicker
    */
   @Prop({ reflect: true }) allowEmpty = false;
 
+  /**
+   * When true, the color picker will process and display alpha characters.
+   */
+  @Prop() opacityEnabled = false;
+
+  @Watch("opacityEnabled")
+  handleOpacityEnabledChange(opacityEnabled: boolean): void {
+    const { format } = this;
+
+    if (opacityEnabled && format !== "auto" && !alphaCompatible(format)) {
+      console.warn(
+        `ignoring opacityEnabled as the current format (${format}) does not support alpha`
```

Comment:
Should we have some sort of logging utility for logging out information specific to a component instance? I'm wondering if users will see this warning and know where its coming from.

Reply:
Good point. Maybe we also need to consider a flag as this message is more for developers and not end-users.